managed/services/agents/registry.go

```diff
@@ -208,7 +213,14 @@ func (r *Registry) register(stream agentpb.Agent_ConnectServer) (*pmmAgentInfo,
 	return agent, nil
 }
 
-func (r *Registry) authenticate(md *agentpb.AgentConnectMetadata, q *reform.Querier) (*models.Node, error) {
+func (r *Registry) authenticate(ctx context.Context, md *agentpb.AgentConnectMetadata, q *reform.Querier) (*models.Node, error) {
```

Comment:
do we have any tests for this method? can we add a few unit tests?

Reply:
OK I will add them.